### PR TITLE
why3: 1.3.1 -> 1.3.2

### PR DESCRIPTION
**Pull request by yrashk**
###### Motivation for this change

New version

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by drewrisinger — CHANGES_REQUESTED** [review]
* [x] Diff LGTM as-is
* [ ] Suggestion: add ``meta.changelog = "http://why3.lri.fr/doc/changes.html";``
* [ ] Suggestion: change to pull source from Inria Gitlab (see comment)
* [x] Commits LGTM
* [x] Builds via ``nix-review``:

```
https://github.com/NixOS/nixpkgs/pull/97241
2 packages built:
framac why3
```

File: pkgs/applications/science/logic/why3/default.nix [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 stdenv.mkDerivation {
   pname = "why3";
-  version = "1.3.1";
+  version = "1.3.2";
 
   src = fetchurl {
-    url = "https://gforge.inria.fr/frs/download.php/file/38291/why3-1.3.1.tar.gz";
-    sha256 = "16zcrc60zz2j3gd3ww93z2z9x2jkxb3kr57y8i5rcgmacy7mw3bv";
+    url = "https://gforge.inria.fr/frs/download.php/file/38366/why3-1.3.2.tar.gz";
```
[the remark this answers is not in the document]
Comment (danieldk) [reply pkgs/applications/science/logic/why3/default.nix:9]: Even better, `fetchFromGitLab` should also work:

```
fetchFromGitLab {
  domain = "gitlab.inria.fr";
  // The usual attributes.
};
```

[the remark this answers is not in the document]
Comment (drewrisinger) [reply pkgs/applications/science/logic/why3/default.nix:9]: Learned something new!

**Comment by vbgl** [comment]
Subsumed by #98441

**Pull request closed without merging**